ansible/examples/centos8-vagrant.yml

```diff
@@ -0,0 +1,57 @@
+---
+## This playbook force reinstalls vagrant and the vagrant-libvirt plugin on CentOS 8.  It assumes:
+##   - You've run slave.yml
+##   - You have `html2text` installed locally and it's in your path
+##
+## Example:
+## ansible-playbook -vvv centos8-vagrant.yml --limit braggi21*
+
+- hosts: all
+  become: true
+  user: ubuntu
+  vars:
+    jenkins_user: jenkins-build
+
+  tasks:
+    - name: Get the latest vagrant version
+      shell: curl -s https://releases.hashicorp.com/vagrant/ | html2text | grep '*' | grep -v '../' | head -n 1 | cut -d '_' -f2
```

Comment:
```bash
$ curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/hashicorp/vagrant/tags | jq '.[0]["name"]'
"v2.2.10"
$
```

not sure whether this is 100% sure the first element is always the latest tag though, to be double checked...

Reply:
There's no guarantee the repo will be available at `releases.hashicorp.com/vagrant` though.

Reply:
yeh that's kind of detail i'm not 100% sure